mesonbuild/minstall.py

```diff
@@ -792,6 +793,8 @@ def run(opts: 'ArgumentType') -> int:
     log_dir = os.path.join(private_dir, '../meson-logs')
     if not os.path.exists(os.path.join(opts.wd, datafilename)):
         sys.exit('Install data not found. Run this command in build directory root.')
+    b = build.load(opts.wd)
+    setup_vsenv(b.need_vsenv)
```

Comment:
Shouldn't you do that inside the not no_rebuild condition below?

Reply:
Good point!